pycsw/core/util.py

```diff
@@ -284,7 +288,7 @@ def transform_mappings(queryables, typename, reverse=False):
     if reverse:  # from csw:Record
         for qbl in queryables.keys():
             if qbl in typename.values():
-                tmp = [k for k, v in typename.iteritems() if v == qbl][0]
+                tmp = [k for k, v in typename.items() if v == qbl][0]
```

Comment:
I am unfamiliar with the code to know if an un-ordered dict is an issue here.  I will try and let's hope the tests a comprehensive enough.